src/QuokkaSimulation.hpp

```diff
@@ -123,7 +123,8 @@ template <typename problem_t> class QuokkaSimulation : public AMRSimulation<prob
 	static constexpr int nstartHyperbolic_ = RadSystem<problem_t>::nstartHyperbolic_;
 
 	amrex::Real radiationCflNumber_ = 0.3;
-	int maxSubsteps_ = 10; // maximum number of radiation subcycles per hydro step
+	int maxSubsteps_ = 10;				// maximum number of radiation subcycles per hydro step
+	amrex::Real dustGasInteractionCoeff_ = 2.5e-34; // erg cm^3 s^−1 K^−3/2
```

Comment:
Can we make this something that is set as a physical constant instead of hardwired? We might want to do tests varying it or do tests with different unit systems.

Reply:
Never mind, I see that you're just setting a default and that you do read it later.

Reply:
That's exactly why I set it inside `QuokkaSimulation.hpp` and control it with a parameter file. You can see that below this line `dustGasInteractionCoeff_` is read from the parameter file as `radiation.dust_gas_interaction_coeff`. In the RadDust test, which uses a dimensionless unit system, it is set to 10^6.

Reply:
That number is there as a default values in CGS units.